k8s/namespaces/pipeline.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pipeline
+
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: pipeline-quota
+  namespace: pipeline
+spec:
+  hard:
+    limits.cpu: 2000
+    limits.memory: 4Gi
+    resourcequotas: 1
```

Comment:
Hi @123MwanjeMike this might not be enough especially incase we are performing batch streaming.

Reply:
How about 4vCPU and 8GB RAM?

Reply:
Updated to 4vCPU and 8GB RAM